include/msgpack/v1/detail/cpp11_zone.hpp

```diff
@@ -25,153 +25,108 @@ namespace msgpack {
 /// @cond
 MSGPACK_API_VERSION_NAMESPACE(v1) {
 /// @endcond
-
+    
```

Comment:
Please remove redundant whitespaces.

Reply:
Originally I tried to make as few changes to the original as possible, in terms of formatting. But maybe I should also bring the other parts of this file to a common style, like moving '{' to another line and empty lines between methods? And now it's all kind of chaotic.